addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -228,9 +228,8 @@ function colorElement(element, color, mode) {
         .replace(/\btext-gradient\b/g, '') // cannot be combined with setting a background
         .replace(/\s+/, ' ');
     element.className !== newClassName && (element.className = newClassName);
-    element.style['background-image'] = '';
     if (mode === 'backgroundColor') {
-        element.style['background'] = '';
+        element.style['background-image'] = '';
```

Comment:
@bso-odoo Could you have a look at this PR please ? I have a feeling that the suggested fix is wrong but I am not familiar with gradients so I'm not so sure. The removal of `element.style['background-image'] = '';` feels correct because I suppose it would otherwise remove the gradient and so that is the core of the fix. However, the change from `background` to `background-image` here seems fishy to me. Now that being said, I suppose a full plain color can also work as a `background-image` so it probably doesn't break anything... and in fact it's probably better to avoid having a gradient defined as `background-image` _and_ a plain color defined as `background`... so it's better to define both types of "colors" as `background-image`. In fact... this PR might be correct. I'm really unsure about this one. What do you think ?

Reply:
@bso-odoo Thank you for the detailed explanation ! I'm going to ping PDE and DBO on the task.

Reply:
Note that we could work on a way support gradient over gradient... Nesting several `font` elements, one about the background and one about the foreground actually produce the expected visual effect... but I do not think the editor currently expects this kind of content, nor what would be the required effort to achieve this.

E.g.:
```xml
<p class="lead" data-original-title="" title="" aria-describedby="tooltip676830">
    <font style="background-image: linear-gradient(135deg, rgb(255, 204, 51) 0%, rgb(226, 51, 255) 100%);" data-original-title="" title="" aria-describedby="tooltip490548">
        <font class="text-gradient" style="background-image: linear-gradient(135deg, rgb(47, 128, 237) 0%, rgb(178, 255, 218) 100%);">
            This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.
        </font>
    </font>
</p>
```
![image](https://user-images.githubusercontent.com/71644421/213487405-d63e211b-483d-4689-a89f-a8c62d20a63f.png)